lib/iris/analysis/__init__.py

```diff
@@ -1584,27 +1584,19 @@ def _lazy_max_run(array, axis=-1, **kwargs):
 
 
 def _rms(array, axis, **kwargs):
-    # XXX due to the current limitations in `da.average` (see below), maintain
-    # an explicit non-lazy aggregation function for now.
```

Comment:
I'm guessing this comment about explicit non-lazy function is because previously some aggregators used a dask function for both lazy and non-lazy cases.  We had a rethink on that at #4905.